- üëã Hi, I‚Äôm Lucas Rincon de la Rosa
- üëÄ I‚Äôm interested in cool stuff like machine learning and deep learning
- Strong defender of Open Science and Open source code
- Currently PhD student in Bioinformatics
- Computer vision internship in image segmentation
- üå± M2 master by Sorbonne Universit√© in Paris in Machine Learning and AI

# About me
Currently PhD student at the Paris Brain Institute (ICM), specializing in Genetics and Tumor Development. My research primarily focuses on Primary Central Nervous Lymphomas (PCNSL), where I leverage both image and omics data. Drawing on my in mathematics and computer science, I employ applied AI techniques to address challenges in patient diagnosis. My role at the ICM does not only fulÔ¨Åll my passion for research and applied mathematics but also provided me with invaluable opportunities to apply these skills to the dynamic and important landscape of medical healthcare.

# Research

*Title: Models for Gliomas Histopathology Image Classification.*<br>
Abstract: Abstract: Diffusely infiltrating gliomas in adults are currently classified into WHO grades I‚ÄìIV, reflecting varying malignancy levels. Recent advances have led to a refined classification of diffusely infiltrating gliomas in adults into three groups: IDH-mutant, 1p/19q codeleted tumors (best prognosis), IDH-mutant, 1p/19q non-codeleted tumors (intermediate prognosis), and IDH wild-type tumors (poor prognosis). This study aims to use a deep learning (DL) image classification model on histopathology images from various cohorts to predict glioma grade, IDH status, and 1p/19q codeletion.

*Title: Spatially-resolved transcriptomics meets Deep Learning: denoising omics data matrix using Optimal Transport and Graph Attention Networks.*<br>
Abstract: In this study we introduce a novel deep learning model for denoising spatial transcriptomics RNA sequencing data, leveraging the power of optimal transport and graph attention mechanisms. We called our model Graph Attention with Optimal Transport, Transformers and Time diffusion (GO3T) which combines the mathematical accuracy of optimal transport to compute distance similarities with the dynamic learning capabilities of graph attention networks. This integration effectively mitigates noise and preserves spatial gene expression patterns. To validate our model's performance, we conducted a comprehensive benchmark against state-of-the-art methods such as GraphST, SpaGCN, and STAGATE as well as ScanPy. Our results demonstrate superior clustering metrics, highlighting the model's ability to maintain biological relevance.

# Contact

For more detailed information, please visit my website at [lucas-rdlr.github.io](lucas-rdlr.github.io).
